modules/core/src/v2/internal/tssUtils.ts

```diff
@@ -285,4 +340,153 @@ export class TssUtils {
 
     return keychains;
   }
+
+  async signTSSTxRequest(
+    unsignedTx: TransactionRequestJSON,
+    walletPassphrase: string,
+    reqId: RequestTracer
+  ): Promise<any> {
+    const { txRequestId } = unsignedTx;
+
+    const signablePayload = await this.getSignablePayload(unsignedTx);
```

Comment:
change to `const signablePayload = Buffer.from(unsignedTx.signableHex, 'hex');`

we can expect validation to be done _before_ this function is called.